### PR TITLE
Feat/i18n

### PR DESCRIPTION
**Pull request by quassy**

### CODE REVIEW

File: index.html
Comment (jamesjoyce92) [inline index.html:1]: somehow the text is not shown anymore

Comment (quassy) [reply index.html:1]: I cannot see the screenshot

Comment (jamesjoyce92) [reply index.html:1]: tested it in chrome, and didn't get the artist/title and text anymore

Comment (jamesjoyce92) [reply index.html:1]: advent

Comment (quassy) [reply index.html:1]: Can't reproduce :(

Is the cache cleared and did you start the page locally with a server like `python3 -m http.server --directory .` and then `localhost:8000`? There are sometimes issues with CORS when using `file://` urls, that affect loading anything with JS.

**Pull request merged**